package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "storyblok",
-  "version": "3.15.0",
+  "version": "4.0.0",
```

Comment:
Please stay with the 3.15 or add 3.25, I know it's not the best case but we're not making a new version.

Reply:
```suggestion
  "version": "3.25.0",
```

Reply:
I still think this should be considered a 'Breaking' change and thus requires a major version (actually in the previous commit already), but for now i have updated it to 3.25.0